### PR TITLE
Add at-risk features (#662).

### LINKED ISSUE #662: Compile and list at-risk features in SoTD.
The following features are currently marked for or should be marked for at-risk. Please edit this comment as needed to add additional features.

- [x] ``#animation-version-2``
- [x] ``#bpd``
- [x] ``#display-inlineBlock``
- [x] ``#extent-measure``
- [x] ``#font``
- [x] ``#fontSelectionStrategy-character``
- [x] ``#ipd``
- [x] ``#rubyAlign-withBase``
- [x] ``#textOrientation-sideways-LR``
- [ ] ``#validation``

For each at-risk feature, there should be a feature designator and an entry in the SoTD citing that designator. Checking one of the above means that these conditions are satisfied in some PR.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #662.

### CODE REVIEW

File: spec/status.xml [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
 <p>For this specification to exit CR, the following criteria must be met. For each feature introduced in this specification:</p>
 <olist>
-<item><p>At least two independent implementations must support the feature. If both transformation semantics and presentation semantics are defined for the feature,
-then at least one implementation must be a transformation processor and at least one implementation must be a presentation processor;</p></item>
-<item><p>At least one of those implementations must be a validating processor that successfully verifies valid or invalid use of the feature as applicable.</p></item>
+  <item><p>At least two independent implementations must support the feature. If both transformation semantics and presentation semantics are defined
+  for the feature, then at least one implementation must be a transformation processor and at least one implementation must be a
+  presentation processor;</p></item>
+  <item><p>At least one of those implementations must be a validating processor that successfully verifies valid or invalid use of the
+  feature as applicable.</p></item>
```
Comment (nigelmegitt) [inline spec/status.xml]: Seems like some editor has line wrapped this and added indentation, contrary to normal practice in the TTML spec. No editorial change here.

File: spec/status.xml [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+<ulist>
+<item><p><loc href="#feature-animation-version-2">#animation-version-2</loc></p></item>
+<item><p><loc href="#feature-border-radii">#border-radii</loc></p></item>
+<item><p><loc href="#feature-bpd">#bpd</loc></p></item>
+<item><p><loc href="#feature-display-inlineBlock">#display-inlineBlock</loc></p></item>
+<item><p><loc href="#feature-extent-measure">#extent-measure</loc></p></item>
+<item><p><loc href="#feature-font">#font</loc></p></item>
+<item><p><loc href="#feature-fontSelectionStrategy-character">#fontSelectionStrategy-character</loc></p></item>
+<item><p><loc href="#feature-rubyAlign-withBase">#rubyAlign-withBase</loc></p></item>
+<item><p><loc href="#feature-textOrientation-sideways-LR">#textOrientation-sideways-LR</loc></p></item>
```
Comment (nigelmegitt) [inline spec/status.xml]: Please add `#validation`, which I consider to be at risk because of being introduced very recently and because there is very weak consensus to include `ttp:validation` and `ttp:validationAction`.

File: spec/status.xml (hunk 1 above)
Comment (skynavga) [reply spec/status.xml]: Are you withholding approval on this basis? Source code style is the prerogative of the editor(s) since there is no WG standard.

File: spec/status.xml [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
+</ulist>
+<p>“At-risk” is a W3C Process term-of-art, and does not necessarily
+imply that the feature is in danger of being dropped or delayed. It
+means that the Working Group believes the feature may have difficulty being
+interoperably implemented in a timely manner, and marking it as such
+allows the Working Group to drop the feature if necessary when transitioning to
+the Proposed Rec stage, without having to publish a new Candidate Rec
```
Comment (nigelmegitt) [inline spec/status.xml]: Reads a little awkwardly. I suggest rewording the last clause to ", without first having to publish a new Candidate Rec omitting the feature."

File: spec/ttml2.xml [hunk 4]
```diff
@@ -21189,6 +21201,21 @@ attribute.</p>
...
 attribute.</p>
+<p>Notwithstanding the above, support for the <code>#textOrientation</code> feature does not require support for the
+<loc href="#feature-textOrientation-sideways-LR"><code>#textOrientation-sideways-LR</code></loc> feature.</p>
+</div3>
+<div3 id="feature-textOrientation-sideways-LR">
+<head>#textOrientation-sideways-LR</head>
+<p>A TTML <loc href="#terms-transformation-processor">transformation processor</loc> supports the
+<code>#textOrientation-sideways-LR</code> feature if it recognizes and is capable of
+transforming the <code>sidewaysLeft</code> and <code>sidewaysRight</code> values of the <loc
+href="#style-attribute-textOrientation-sideways-LR"><att>tts:textOrientation-sideways-LR</att></loc>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: Incorrect attribute name - the href should be `#style-attribute-textOrientation` and the attribute name is `tts:textOrientation`

File: spec/ttml2.xml [hunk 5]
```diff
@@ -21189,6 +21201,21 @@ attribute.</p>
...
+attribute.</p>
+<p>A TTML <loc href="#terms-presentation-processor">presentation processor</loc> supports the
+<code>#textOrientation-sideways-LR</code> feature if it implements presentation semantic support
+for the <code>sidewaysLeft</code> and <code>sidewaysRight</code> values of the <loc
+href="#style-attribute-textOrientation-sideways-LR"><att>tts:textOrientation-sideways-LR</att></loc>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: As above, incorrect attribute name - the href should be `#style-attribute-textOrientation` and the attribute name is `tts:textOrientation`

File: spec/status.xml (hunk 3 above)
Comment (skynavga) [reply spec/status.xml]: I copied this out of a CSS CR, but will address.

File: spec/status.xml (hunk 2 above)
Comment (skynavga) [reply spec/status.xml]: Fixed.

File: spec/status.xml (hunk 3 above)
Comment (skynavga) [reply spec/status.xml]: Fixed.

File: spec/ttml2.xml (hunk 4 above)
Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

File: spec/ttml2.xml (hunk 5 above)
Comment (skynavga) [reply spec/ttml2.xml]: Fixed.

File: spec/status.xml (hunk 1 above)
Comment (nigelmegitt) [reply spec/status.xml]: @skynavga Huh? No, just pointing out that the diff looks like there's a change here, but in practice there is none.

Comment (skynavga) [reply spec/status.xml]: ok
